src/XMakeTasks/UnitTests/Copy_Tests.cs

```diff
@@ -1845,6 +1846,46 @@ public CopyNotHardLink_Tests()
         }
     }
 
+    public class CopyHardAndSymbolicLink_Tests : Copy_Tests
```

Comment:
Does this need to inherit from `Copy_Tests`? It contains only a single test and doesn't need to run all of the copy tests in a different configuration.